lib/chef/win32/registry.rb

```diff
@@ -293,7 +293,11 @@ def node
       end
 
       def machine_architecture
-        node[:kernel][:machine].to_sym
+        if node.nil?
+          ::RbConfig::CONFIG["target_cpu"] == "x64" ? :x86_64 : :i386
+        else
```

Comment:
would we ever need this else?

Reply:
I added that because I didn't want to assume we were ruling out 32-bit anything right now